cellbrowser_tools/fov_processing.py

```diff
@@ -568,31 +835,36 @@ def generate_and_save(self, do_segmented_cells=True):
 
             # for bn in cell_meta.bounds:
             #    print(bn, cell_meta.bounds[bn])
+            minz = int(bounds[2][0])
+            maxz = int(bounds[2][1])
+
+            # print(f"cell Z size = {cropped.shape[1]}")
+            # print(f"cell Z bounds: {minz} to {maxz}")
 
             # copy self.omexml for output
             copyxml = None
 
             log.info("making cropped image...")
-            copied = copy.deepcopy(self.omexml.dom)
-            copyxml = OMEXML(rootnode=copied)
+            # start with original metadata and make a copy
+            # copyxml = self.omexml.copy(deep=True)
+            copystr = to_xml(self.omexml)
+            copyxml = from_xml(copystr)
```

Comment:
for some reason the commented-out line that is supposed to copy the object never worked.  the to_xml and from_xml does the job for now.  I need to follow up on this.